Segregation

renanxcortes opened the pull request:
Serge, I added more segregation measures in the function. The main objective was to check how each measure would behave in a 'non-segregated' society.

renanxcortes committed:
adding more different measure

This commit adds several segregaton measures and even create a "Centralized Isolation" not covered in Masseys paper in order to build a Overall Segregation measure that the value assumes zero in a uniform distribution (non-segregated society).

renanxcortes committed:
add the new function to the .py file

This commit adds several segregaton measures and even create a "Centralized Isolation" not covered in Masseys paper in order to build a Overall Segregation measure that the value assumes zero in a uniform distribution (non-segregated society).

renanxcortes committed:
including a prototype of permutation inference notebook

It's still not optimized, but the idea might be with this approach

renanxcortes committed:
reducing number of permutations in the notebook

just reduced the number of permutations to avoid overloading

renanxcortes committed:
renaming notebook name

adding underscores in the simulation notebook

renanxcortes commented:
I also added a notebook with a prototype for the inference framework for the first approach.

renanxcortes committed:
detailing the construction of the segregation index and adding a inference framework for segregation in Riverside

renanxcortes committed:
adding new functions to the .py archive

renanxcortes committed:
adding potential inference for two different counties

sjsrey commented:
It is probably a good idea to keep the datasets out of this repository since they are going to eat up the storage capacity and git tracking (3k file changes is a lot to keep track of ;-> )  One thing we can do is add a link to where the data are and have the users/devs have a local copy of the data for local testing.

renanxcortes committed:
removed supplementry files and iteration name parameter

I removed the data files and json files and put them in another repo and changed the path for a local repo and highlights this at the beggining of each notebook. I also changed the 'permutations' parameter name to 'iterations'

renanxcortes committed:
adding more uneveness measures and inference framework for one space-time sample

Adding the Gini coefficient measure and the entropy. In addition, it's possibly to build the inference framework assuing eveness, spatial permutation and spatial permutation with eveness with the null_approah argument

renanxcortes committed:
adding condition on pi and changing to random labelling

This commit comprehends two things: firstly, I've seen that when a specific tract presented 0 for total population, some intermediate calculations (specially for the Gini index) the return of nan breaks the code, so I changed to a np.where statements that changes to zero when this happens,. Secondly, I changed the inference framewor to compare two cities for the random labelling approach.

renanxcortes committed:
including random data approach

renanxcortes committed:
adding random data permutation framework

renanxcortes committed:
adding inference framework for the same county between two points in time

renanxcortes committed:
changing to 38021 tract as default

renanxcortes committed:
including table of contents in both notebooks
